django_elasticsearch_dsl/documents.py

```diff
@@ -156,11 +156,19 @@ def parallel_bulk(self, actions, **kwargs):
         # the result is currently not used upstream anyway.
         return (1, [])
 
+    def generate_id(self, object_instance):
```

Comment:
It is better to have static method so it can be called directly without initiating the object. Like `Document.generate_id(model_object)`